templates/navigation/ys-breadcrumb-block.html.twig

```diff
@@ -0,0 +1,7 @@
+{{ attach_library('atomic/breadcrumbs' ) }}
+
+{# only displays breadcrumbs if 3 lines deep #}
+
+{% if items.2 %}
+{% include '@organisms/menu/breadcrumbs/breadcrumbs.twig' %}
+{% endif %}
```

Comment:
This is the method I was thinking we'd use, but it should be done in the component library so that it applies to all future themes. So, you should be able to do the same thing around lines 15 to the end: https://github.com/yalesites-org/component-library-twig/blob/develop/components/03-organisms/menu/breadcrumbs/breadcrumbs.twig

You'll just need to use the `breadcrumbs__items` variable, instead of `items`